frontend/packages/kubevirt-plugin/integration-tests/tests/vm.migration.scenario.ts

```diff
@@ -27,21 +29,26 @@ describe('Test VM Migration', () => {
   const MIGRATE_VM = 'Migrate Virtual Machine';
   const CANCEL_MIGRATION = 'Cancel Virtual Machine Migration';
   const VM_BOOT_AND_MIGRATE_TIMEOUT = VM_BOOTUP_TIMEOUT_SECS + VM_MIGRATION_TIMEOUT_SECS;
+  const STORAGE_ACCESS_MODE = resolveStorageDataAttribute(kubevirtStorage, 'accessMode');
```

Comment:
please make this lowercase as it is resolved dynamically and is not a real constant